backend/src/modules/mobile/mobile.controller.ts

```diff
@@ -132,6 +137,44 @@ export class MobileController {
     return user;
   }
 
+  @Post('/lock')
+  @HttpCode(201)
+  lock(@Body() data: LockDeviceDTO) {
+    const message = {
+      command: Number(data.status)
+    }
+    const result = this.MqttService.publishMessage("lock", data.device, JSON.stringify(message));
+    return result ? 'success' : 'failure';
+  }
+
+  @Post('/motionThreshold')
```

Comment:
is this to change the motion threshold? if so can we change it to reflect it more clearly, like maybe `/changeMotionThreshold` or `/motionThreshold/update`